ide/configwin.ml

```diff
@@ -37,8 +37,10 @@ type return_button =
   | Return_cancel
 
 let string = Configwin_ihm.string
+(*
 let strings = Configwin_ihm.strings
 let list = Configwin_ihm.list
+*)
```

Comment:
Why keep this commented?

Reply:
If the outcome of the discussion is that the user query and wizard tactics preferences are not worth being preserved and/or that noone is motivated to adapt the code and use the recommanded `tree_store` feature in place of the current `clist`, I guess that the corresponding code, become dead, can indeed be removed completely.

Otherwise said, until there is some visibility on what this code will become, I preferred to keep it in the files at the current stage and to ask ourselves this question again before merging.